.rustfmt.toml

```diff
@@ -10,15 +10,46 @@ reorder_imports = true
 reorder_modules = true
 tab_spaces = 4
 use_field_init_shorthand = true
+use_small_heuristics = "Default"
 use_try_shorthand = true
-edition = "2018"
 
-# unstable options (for one-off fixes and CI)
-merge_imports = true
-wrap_comments = true
+# unstable options
+binop_separator = "Front"
+blank_lines_lower_bound = 0
+blank_lines_upper_bound = 1
+brace_style = "PreferSameLine"
+combine_control_expr = false
 comment_width = 100
 condense_wildcard_suffixes = true
+control_brace_style = "AlwaysSameLine"
+empty_item_single_line = true
+enum_discrim_align_threshold = 50
```

Comment:
`0` is the default value
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#enum_discrim_align_threshold

Ok, so `50` is fairly arbitrary, but my rationale for choosing that value was something large enough so that most of our code would line up nicely like in the `enum Foo` example for a value of `20`.  My goal was to have everything line up vertically, and I figured that the vast majority of our enum variants fit within 50 characters.